python/mxnet/contrib/onnx/onnx2mx/import_onnx.py

```diff
@@ -157,15 +156,15 @@ def get_graph_metadata(self, graph):
                    }
         return metadata
 
-    def graph_to_gluon(self, graph, context):
+    def graph_to_gluon(self, graph, ctx):
```

Comment:
Thanks for this fix as we're already using ctx everywhere in gluon.

Reply:
No problem. Thank you for reviewing! I appreciate the merge.